packages/popup/src/pages/Home/components/AddToken.js

```diff
@@ -89,6 +101,9 @@ function AddToken({onClose, onOpen}) {
       if (result) {
         mutate([WALLETDB_REFETCH_BALANCE])
         mutate([WALLETDB_ADD_TOKEN_LIST])
```

Comment:
the second mutate should be invoked after the first mutate resolved

```js
mutate([WALLETDB_REFETCH_BALANCE]).then(() => mutate([WALLETDB_ADD_TOKEN_LIST]))
```

Reply:
done